libs/dataset-explorer/src/lib/TableView/TableView.tsx

```diff
@@ -0,0 +1,294 @@
+    if (
+      this.props.modelType &&
+      this.props.modelType === ModelTypes.Regression
+    ) {
+      // don't use groups since there are no correct/incorrect buckets
+      this.props.selectedCohort.cohort.sort();
+    } else {
+      this.props.selectedCohort.cohort.sortByGroup(
+        JointDataset.IndexLabel,
+        (row) =>
+          row[JointDataset.TrueYLabel] === row[JointDataset.PredictedYLabel]
+      );
+      // find first incorrect item
+      const firstIncorrectItemIndex =
+        this.props.selectedCohort.cohort.filteredData.findIndex(
+          (row) =>
+            row[JointDataset.TrueYLabel] !== row[JointDataset.PredictedYLabel]
+        );
+      const noIncorrectItem = firstIncorrectItemIndex === -1;
+
+      groups = [
+        {
+          count: noIncorrectItem
+            ? this.props.selectedCohort.cohort.filteredData.length
+            : firstIncorrectItemIndex,
+          isCollapsed: true,
+          key: "groupCorrect",
+          level: 0,
+          name: localization.ModelAssessment.FeatureImportances
+            .CorrectPredictions,
+          startIndex: 0
+        },
+        {
+          count: noIncorrectItem
+            ? 0
+            : this.props.selectedCohort.cohort.filteredData.length -
+              firstIncorrectItemIndex,
+          key: "groupIncorrect",
+          level: 0,
+          name: localization.ModelAssessment.FeatureImportances
+            .IncorrectPredictions,
+          startIndex: firstIncorrectItemIndex
+        }
+      ];
+    }
+
+    const cohortData = this.props.selectedCohort.cohort.filteredData;
+    const numRows: number = cohortData.length;
+    const indices = this.props.selectedCohort.cohort.filteredData.map(
+      (row: { [key: string]: number }) => {
+        return row[JointDataset.IndexLabel] as number;
+      }
+    );
+
+    const rows = constructRows(
+      cohortData,
+      this.props.jointDataset,
+      numRows,
+      () => false, // don't filter any items
+      indices
+    );
+
+    const numCols: number = this.props.jointDataset.datasetFeatureCount;
+    const featureNames: string[] = this.props.features;
+    const viewedCols: number = Math.min(numCols, featureNames.length);
+    const columns = constructCols(
+      viewedCols,
+      featureNames,
+      this.props.jointDataset,
+      false
+    );
+
+    return {
+      columns,
+      groups,
+      rows
+    };
+  }
+
+  private onRenderDetailsHeader: IRenderFunction<IDetailsHeaderProps> = (
+    props,
+    defaultRender
+  ) => {
+    if (!props) {
+      return <div />;
+    }
+    const onRenderColumnHeaderTooltip: IRenderFunction<IDetailsColumnRenderTooltipProps> =
+      (tooltipHostProps) => <TooltipHost {...tooltipHostProps} />;
+    return (
+      <div>
+        {defaultRender?.({
+          ...props,
+          onRenderColumnHeaderTooltip,
+          selectAllVisibility: SelectAllVisibility.hidden
+        })}
+      </div>
+    );
+  };
```

Comment:
Could you please create onRenderDetailsHeader.tsx and move the component there? Thanks

Reply:
done!